### PR TITLE
New header and footers

### PR DESCRIPTION
**Pull request by MichaelWalker**
Header and footer on invest pages now uses the new international header and footer from directory components.

There seems to be some issues with the VPN this morning, so I've not been able to update the Vault with the new environment variables! Will do that once I can get in!

image

image

### CODE REVIEW

File: requirements.in [hunk 1]
```diff
@@ -12,9 +12,8 @@ jsonschema==2.6.0
 django-recaptcha==1.2.0
 directory-cms-client==8.2.0
 directory-validators==5.0.0
-directory-constants==13.1.0
-directory-components==6.1.0
-export-elements==2.0.0
+directory-constants==14.2.0
+directory-components==7.1.2
```
Comment (eadpearce) [inline requirements.in]: may as well update to most recent version here

Comment (MichaelWalker) [reply requirements.in]: done - @eadpearce are you happy for this to be merged now?

**Comment by kaedroho** [comment]
This PR removes the language switcher, is this coming back?

**Comment by MichaelWalker** [comment]
@kaedroho - yes! 

I believe @eadpearce is working (/will soon be working?) on adding the language switcher to the new header and footer.

**Pull request merged**